### PR TITLE
conduktor: init at 2.15.1

### PR DESCRIPTION
**Pull request by trobert**
###### Motivation for this change
Add conduktor (https://www.conduktor.io/)

Conduktor is a GUI over the Kafka ecosystem, to make the development and management of Apache Kafka clusters as easy as possible.

Free of charge for a single broker.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/conduktor/default.nix [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{ stdenv, fetchurl, jdk11, unzip, makeWrapper, makeDesktopItem, copyDesktopItems }:
+
+stdenv.mkDerivation rec {
+  pname = "conduktor";
+  version = "2.7.2";
+  src = fetchurl {
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/conduktor/default.nix:6]:
```suggestion
  version = "2.7.2";

  src = fetchurl {
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please change the PR title and commit message to ``conduktor: init at 2.7.2``

File: pkgs/applications/misc/conduktor/default.nix [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    runHook postInstall
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Apache Kafka Desktop Client";
+    longDescription = ''
+      Conduktor is a GUI over the Kafka ecosystem, to make the development
+      and management of Apache Kafka clusters as easy as possible.
+    '';
+    homepage = https://www.conduktor.io/;
```
Comment (roberth) [inline pkgs/applications/misc/conduktor/default.nix:48]:
```suggestion
    homepage = "https://www.conduktor.io";
```

File: pkgs/applications/misc/conduktor/default.nix [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    changelog = "https://www.conduktor.io/changelog/#${version}";
+    license = stdenv.lib.licenses.unfree;
+    maintainers = with maintainers; [ trobert ];
+    platforms = platforms.linux ++ platforms.darwin;
```
Comment (roberth) [inline pkgs/applications/misc/conduktor/default.nix:52]: Could you make `src` conditional on the platform? Does the linux zip provide support for multiple cpu types?

Comment (trobert) [reply pkgs/applications/misc/conduktor/default.nix:52]: the zip is not compatible with darwin (it's a specific .pkg binary) so I'll drop darwin for now, as I don't have a mac to test.

File: pkgs/applications/misc/conduktor/default.nix [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+  meta = with lib; {
+    description = "Apache Kafka Desktop Client";
+    longDescription = ''
+      Conduktor is a GUI over the Kafka ecosystem, to make the development
+      and management of Apache Kafka clusters as easy as possible.
+    '';
+    homepage = "https://www.conduktor.io/";
+    changelog = "https://www.conduktor.io/changelog/#${version}";
+    license = stdenv.lib.licenses.unfree;
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/conduktor/default.nix:50]:
```suggestion
    license = licenses.unfree;
```

File: pkgs/applications/misc/conduktor/default.nix [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    sha256 = "1hkhhwh3ihsqqwknlrpj49s10qvsnaxp04xrh2yys3zya707ws9g";
+  };
+
+  nativeBuildInputs = [ unzip makeWrapper copyDesktopItems ];
+
+  icon = fetchurl {
+    url = "https://github.com/conduktor/builds/raw/v${version}/.github/resources/Conduktor.png";
+    sha256 = "0s7p74qclvac8xj2m22gfxx5m2c7cf0nqpk5sb049p2wvryhn2j4";
+  };
+
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/conduktor/default.nix:18]:
```suggestion
```

File: pkgs/applications/misc/conduktor/default.nix [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+  desktopItems = makeDesktopItem {
+    type = "Application";
+    name = pname;
+    desktopName = "Conduktor";
+    genericName = meta.description;
+    exec = pname;
+    icon = icon;
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/conduktor/default.nix:25]:
```suggestion
    icon = fetchurl {
      url = "https://github.com/conduktor/builds/raw/v${version}/.github/resources/Conduktor.png";
      sha256 = "0s7p74qclvac8xj2m22gfxx5m2c7cf0nqpk5sb049p2wvryhn2j4";
    };
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/top-level/all-packages.nix [hunk 7]
```diff
@@ -3706,6 +3706,8 @@ in
 
   conda = callPackage ../tools/package-management/conda { };
 
+  conduktor = callPackage ../applications/misc/conduktor { } ;
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:3709]:
```suggestion
  conduktor = callPackage ../applications/misc/conduktor { };
```

File: pkgs/applications/misc/conduktor/default.nix [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+stdenv.mkDerivation rec {
+  pname = "conduktor";
+  version = "2.14.4";
+
+  src = fetchurl {
+    url = "https://github.com/conduktor/builds/releases/download/v${version}/Conduktor-linux-${version}.zip";
+    sha256 = "1hkhhwh3ihsqqwknlrpj49s10qvsnaxp04xrh2yys3zya707ws9g";
+  };
+
+  nativeBuildInputs = [ unzip makeWrapper copyDesktopItems ];
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/conduktor/default.nix:12]:
```suggestion
  src = fetchzip {
    url = "https://github.com/conduktor/builds/releases/download/v${version}/Conduktor-linux-${version}.zip";
    sha256 = "1hkhhwh3ihsqqwknlrpj49s10qvsnaxp04xrh2yys3zya707ws9g";
  };

  nativeBuildInputs = [ makeWrapper copyDesktopItems ];
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
also please squash all package related commits.

File: pkgs/applications/misc/conduktor/default.nix [hunk 9]
```diff
@@ -0,0 +1,52 @@
+{ stdenv, lib, fetchzip, jdk11, unzip, makeWrapper, makeDesktopItem, copyDesktopItems }:
+
+stdenv.mkDerivation rec {
+  pname = "conduktor";
+  version = "2.14.4";
+
+  src = fetchzip {
+    url = "https://github.com/conduktor/builds/releases/download/v${version}/Conduktor-linux-${version}.zip";
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/conduktor/default.nix:8]: Can we easily build from source?

File: pkgs/applications/misc/conduktor/default.nix [hunk 10]
```diff
@@ -0,0 +1,52 @@
+{ stdenv, lib, fetchzip, jdk11, unzip, makeWrapper, makeDesktopItem, copyDesktopItems }:
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/conduktor/default.nix:1]:
```suggestion
{ stdenv, lib, fetchzip, fetchurl, jdk11, unzip, makeWrapper, makeDesktopItem, copyDesktopItems }:
```

this fixes the current eval error.

File: pkgs/applications/misc/conduktor/default.nix (hunk 9 above)
Comment (trobert) [reply pkgs/applications/misc/conduktor/default.nix:8]: it's not open source

File: pkgs/applications/misc/conduktor/default.nix (hunk 10 above)
Comment (trobert) [reply pkgs/applications/misc/conduktor/default.nix]: fixed, thanks

**Comment by Congee** [comment]
not stale

**Review by roberth — APPROVED** [review]

**Pull request merged**